subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -145,40 +145,16 @@ The last set of classes have no external or internal usages and therefore were d
 - `NoOpChangeListener`
 - `DeferredUtil`
 
-[[exec_properties]]
-=== Properties deprecated in Exec task
-
-* The link:{groovyDslPath}/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:execResult[Exec.execResult]
-property has been deprecated.
-Use the link:{groovyDslPath}/org.gradle.api.tasks.Exec.html#org.gradle.api.tasks.Exec:executionResult[Exec.executionResult]
-property instead.
-
 [[java_exec_properties]]
 ==== Properties deprecated in JavaExec task
 
 * The link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:main[main] getters and setters
 in link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html[JavaExec] task have been deprecated.
 Use the link:{groovyDslPath}/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:mainClass[mainClass] property instead.
```

Comment:
Those I had to leave here because `JavaExecSpec` - the interface with deprecations has two more implementations - `DefaultJavaExecSpec` and `JavaExecHandleBuilder` which live in `core` package and do not have access to `JavaExec` class which is used to construct a link to the DSL reference.
Alternatively, I could have extended the DSL for constructing documentation links to accept Strings and pass the fully qualified link to the DSL reference that way, but I thought the current approach will be simpler.